app/controllers/nsm/steps/claim_confirmation_controller.rb

```diff
@@ -1,13 +1,19 @@
 module Nsm
   module Steps
     class ClaimConfirmationController < Nsm::Steps::BaseController
+      skip_before_action :update_viewed_steps, :prune_viewed_steps
+
       def show
         @laa_reference = current_application.laa_reference
       end
 
       def step_valid?
         current_application.submitted? || current_application.provider_updated?
       end
+
+      def current_application
```

Comment:
Should this not be part of the base controller?

Reply:
This is actually overriding the base controller behaviour, which by default loads `current_application` from the local DB. Most of the many child controllers use the default behaviour. I didn't think the exceptions were quite frequent enough to make it worth complicating the inheritance tree or adding a concern just to handle an overridden one-liner.